assets/javascript/app.js

```diff
@@ -0,0 +1,157 @@
+var triviaQuestions = [{
+        question: "What year was the very first model of the iPhone released?",
+        choices: ["A: 2000", "B: 2007", "C: 2002", "D: 2010"],
+        answer: "B"
+    }, {
+        question: "When Michael Jordan played for the Chicago Bulls, how many NBA Championships did he win?",
+        choices: ["A: 6", "B: 2", "C: 5", "D: 1"],
+        answer: "A"
+    }, {
+        question: "In the TV Show 'The Office' which employee did Michael Scott hit with his car?",
+        choices: ["A: Angela", "B: Kelly", "C: Oscar", "D: Meredith"],
+        answer: "D"
+    },
+    {
+        question: "Which mammal has no vocal cords?",
+        choices: ["A: Pig", "B: Elephant", "C: Octopus", "D: Giraffe"],
+        answer: "D"
+    },
+    {
+        question: "Which planet has the most gravity?",
+        choices: ["A: Venus", "B: Jupiter", "C: Mars", "D: Saturn"],
+        answer: "B"
+    },
+    {
+        question: "Which country produces the most coffee in the world?",
+        choices: ["A: Colombia", "B: Mexico", "C: Brazil", "D: Chile"],
+        answer: "C"
+    },
+    {
+        question: "In which body part can you find the femur?",
+        choices: ["A: Leg", "B: Arm", "C: Back", "D: Stomach"],
+        answer: "A"
+    },
+    {
+        question: "Which country invented tea?",
+        choices: ["A: America", "B: China", "C: Germany", "D: Greece"],
+        answer: "B"
+    },
+    {
+        question: "How many Lord of the Rings films are there?",
+        choices: ["A: 1", "B: 4", "C: 3", "D: 7"],
+        answer: "C"
+    },
+    {
+        question: "In what year was the first episode of South Park aired?",
+        choices: ["A: 1997", "B: 1990", "C: 1994", "D: 2000"],
+        answer: "A"
+    }
+];
+var i = 0;
+var triviaQuestion = triviaQuestions[parseInt(i)];
+var time = 30;
+var timer;
+var numberofQuestions = triviaQuestions.length;
+var correctQuestions = 0;
+
+function runTimer() {
+    clearInterval(timer);
+    timer = setInterval(function gameTimer() {
+        time -= 1;
+        $("#timer").html(`<h4>Time Remaning: ${time} Seconds</h4>`);
+        if (time <= 0) {
+            nextQuestion();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['nextQuestion' was used before it was defined. (no-use-before-define)](https://app.codacy.com/app/columbia-fsf-f2019/trivia-game/pullRequest?prid=4555494)